charms/reconciler.py

```diff
@@ -1,16 +1,22 @@
-import charms.contextual_status as status
 import logging
 
 from ops import HookEvent, Object, StoredState, UpdateStatusEvent
 
+import charms.contextual_status as status
 
 log = logging.getLogger(__name__)
 
 
 class Reconciler(Object):
     stored = StoredState()
 
-    def __init__(self, charm, reconcile_function, exit_status=None):
+    def __init__(
+        self,
+        charm,
+        reconcile_function,
+        exit_status=None,
+        custom_events=None,
```

Comment:
This is a `Iterable[ops.BoundEvent]`? right?  I'm not asking for typing (unless you just want to) but just trying to get it solid in my mind

Reply:
That's correct, `custom_events` are Iterable[ops.BoundEvent]`.